components/atoms/forms/AntFormItemWrapper.tsx

```diff
@@ -64,6 +64,11 @@ export default function AntFormItemWrapper({
         }
         return value;
       }}
+      getValueFromEvent={(event) => {
+        return event?.target?.id.startsWith("numberInputText")
```

Comment:
event is required value, remove optional chaining from it (`event?.` -> `event.`)